code/world.dm

```diff
@@ -18,6 +18,7 @@ var/list/map_transition_config = MAP_TRANSITION_CONFIG
 /world/New()
 	log_world("World loaded at [world.timeofday]")
 	map_ready = 1
+	initLimiter()
```

Comment:
Just define `/world/New()` in one of the sound files and call parent before initializing the limiter